ietf/doc/views_stats.py

```diff
@@ -124,14 +124,11 @@ def chart_newrevisiondocevent(request):
 
 #@cache_page(60*15)
 def chart_data_newrevisiondocevent(request):
```

Comment:
Pretty sure this doesn't work as intended, and it's not being used, so lets remove it.

Reply:
removal done in #8210